docs/Public_Nitro_Devnet.md

```diff
@@ -55,3 +55,60 @@ For a deeper dive into deploying with truffle see [here](Contract_Deployment.md)
+| Delayed Inbox       | [0x1fdbbcc914e84af593884bf8e8dd6877c29035a2](https://goerli.etherscan.io/address/0x1fdbbcc914e84af593884bf8e8dd6877c29035a2) |
+| Sequencer Inbox     | [0xb32f4257e05c56c53d46bbec9e85770eb52425d6](https://goerli.etherscan.io/address/0xb32f4257e05c56c53d46bbec9e85770eb52425d6) |
+| Bridge              | [0x9903a892da86c1e04522d63b08e5514a921e81df](https://goerli.etherscan.io/address/0x9903a892da86c1e04522d63b08e5514a921e81df) |
+| Outbox              | [0xFDF2B11347dA17326BAF30bbcd3F4b09c4719584](https://goerli.etherscan.io/address/0xFDF2B11347dA17326BAF30bbcd3F4b09c4719584) |
+
+### Token Bridge
+
+**IMPORTANT**: _Do **not** simply transfer tokens or Ether to any of the addresses below; it will result in loss of funds._
+
+_Users should only interact with the token bridge via dapp interfaces like https://nitro-devnet-bridge.arbitrum.io.
+
+|                       | Goerli / Nitro Devnet                                                                                                 |
+| --------------------- | --------------------------------------------------------------------------------------------------------------------- |
+| L1 Gateway Router     | [0x8BDFa67ace22cE2BFb2fFebe72f0c91CDA694d4b](https://goerli.etherscan.io/address/0x8BDFa67ace22cE2BFb2fFebe72f0c91CDA694d4b) |
+| L2 Gateway Router     | [0xC502Ded1EE1d616B43F7f20Ebde83Be1A275ca3c](https://nitro-devnet-explorer.arbitrum.io/address/0xC502Ded1EE1d616B43F7f20Ebde83Be1A275ca3c)  |
+| L1 ERC20 Gateway      | [0x6336C4e811b2f7D17d45b6241Fd47F2E11621Ffb](https://goerli.etherscan.io/address/0x6336C4e811b2f7D17d45b6241Fd47F2E11621Ffb) |
+| L2 ERC20 Gateway      | [0xf298434ffE691400b932f4b14B436f451F4CED76](https://nitro-devnet-explorer.arbitrum.io/address/0xf298434ffE691400b932f4b14B436f451F4CED76)  |
+| L1 Arb-Custom Gateway | [0x23D4e0D7Cb7AE7CF745E82262B17eb46535Ae819](https://goerli.etherscan.io/address/0x23D4e0D7Cb7AE7CF745E82262B17eb46535Ae819) |
+| L2 Arb-Custom Gateway | [0x7AC493f26EF26904E52fE46C8DaEE247b9A556B8](https://nitro-devnet-explorer.arbitrum.io/address/0x7AC493f26EF26904E52fE46C8DaEE247b9A556B8)  |
+| L1 Weth Gateway       | [0x64bfF696bE6a087A81936b9a2489624015381be4](https://goerli.etherscan.io/address/0x64bfF696bE6a087A81936b9a2489624015381be4) |
+| L2 Weth Gateway       | [0xf10c7CAA33A3360f60053Bc1081980f62567505F](https://nitro-devnet-explorer.arbitrum.io/address/0xf10c7CAA33A3360f60053Bc1081980f62567505F)  |
+| L1 Weth               | [0xb4fbf271143f4fbf7b91a5ded31805e42b2208d6](https://goerli.etherscan.io/address/0xb4fbf271143f4fbf7b91a5ded31805e42b2208d6) |
+| L2 Weth               | [0x96CfA560e7332DebA750e330fb6f59E2269f40Dd](https://nitro-devnet-explorer.arbitrum.io/address/0x96CfA560e7332DebA750e330fb6f59E2269f40Dd)  |
+| OneStepProver0      | [0x91ddccc4cccbb7609e045e10d311712789f2010f](https://goerli.etherscan.io/address/0x91ddccc4cccbb7609e045e10d311712789f2010f) |
```

Comment:
These onestep prover contracts are run as part of the fraud proofs. Maybe they should be in the protocol section, or the misc section?

Reply:
Fine with either, we do list them in the useful addresses page. On the other hand, in Classic we have these as proxies, do we have the same with Nitro?